### PR TITLE
Api600/uplink sets

### PR DESCRIPTION
**Pull request by AnushaY1916**
### Description
Api600 Support for Uplink Sets

### Issues Resolved
NA

### Check List
- [x] New functionality includes testing.
  - [x] All tests pass (`$ rake test`).
- [ ] New functionality has been documented in the README if applicable.
  - [x] New functionality has been thoroughly documented in the examples (please include helpful comments).
- [x] New endpoints supported are updated in the endpoints-support.md file.
- [x] Changes are documented in the CHANGELOG.

### CODE REVIEW

File: spec/unit/resource/api600/c7000/uplink_set_spec.rb [hunk 1]
```diff
@@ -1,9 +1,9 @@
 require 'spec_helper'
 
-RSpec.describe OneviewSDK::API500::C7000::UplinkSet do
+RSpec.describe OneviewSDK::API600::C7000::UplinkSet do
   include_context 'shared context'
 
-  it 'inherits from OneviewSDK::API300::C7000::UplinkSet' do
+  it 'inherits from OneviewSDK::API500::C7000::UplinkSet' do
     expect(described_class).to be < OneviewSDK::API300::C7000::UplinkSet
```
Comment (soodpr) [inline spec/unit/resource/api600/c7000/uplink_set_spec.rb]: it should inherit from 500

File: spec/unit/resource/api600/synergy/uplink_set_spec.rb [hunk 2]
```diff
@@ -1,9 +1,9 @@
 require 'spec_helper'
 
-RSpec.describe OneviewSDK::API500::Synergy::UplinkSet do
+RSpec.describe OneviewSDK::API600::Synergy::UplinkSet do
   include_context 'shared context'
 
-  it 'inherits from OneviewSDK::API300::Synergy::UplinkSet' do
+  it 'inherits from OneviewSDK::API600::Synergy::UplinkSet' do
     expect(described_class).to be < OneviewSDK::API300::Synergy::UplinkSet
```
Comment (soodpr) [inline spec/unit/resource/api600/synergy/uplink_set_spec.rb]: it should inherit from 500

File: spec/unit/resource/api600/c7000/uplink_set_spec.rb (hunk 1 above)
Comment (AnushaY1916) [reply spec/unit/resource/api600/c7000/uplink_set_spec.rb]: updated as per comment

File: CHANGELOG.md [hunk 3]
```diff
@@ -21,6 +21,7 @@ This release adds support to OneView Rest API version 600 for the hardware varia
    - SAS Logical Interconnect
    - Server Hardware
    - Server Profile Template
+   - Uplink Sets
```
Comment (sijeesh) [inline CHANGELOG.md]: Should be Uplink Set

**Pull request merged**